### DISCUSSION TITLE
Removing the "Gift Cards" side menu from admin UI?

### DISCUSSION THREAD

**Discussion by olalonde** [medusajs/medusa#5199] [Q&A]
Is it possible to remove menu items in the admin side nav?

<img width="285" alt="Screenshot 2023-09-25 at 6 14 36 PM" src="https://github.com/medusajs/medusa/assets/416585/2cd28f99-6f41-49d8-8e56-e9d7af6f1d1b">

Or would I have to fork the admin UI package to do this?

**Comment by pevey** [comment]
You could also edit the sidebar in the @medusajs/admin-ui package within your node_modules folder, and then use the "patch-package" package to make sure the changes stick when the admin-ui package is updated or reinstalled.

**Reply by pevey** [reply]
`yarn add -D patch-package`

Then, add this in the scripts section of your package.json:
```json
"scripts": {
      "postinstall": "patch-package",
...
```

Then, edit the files that you want to edit in your node_modules/@medusajs/admin-ui folder.  Then, run npx patch-package @medusajs/admin-ui.  Then, be sure to add the patch files to your repo.  Now, those patches will be applied any time the admin-ui package is installed.

This is what the patch folder looks like:
![Screenshot_2023-08-31_114724](https://github.com/medusajs/medusa/assets/7490308/611597a6-e6f8-4708-a133-c49de7d76851)

https://discord.com/channels/876835651130097704/1146666324060278884

**Reply by valkor** [reply]
Does this  answer still work with new medusa 2.0?

**Reply by BlazejBatko** [reply]
> Does this answer still work with new medusa 2.0?

@valkor have you checked by any chance if it works on 2.0?

**Comment by haricoddle** [comment]
@pevey can you specify which file to change, I need to hide the product option.

**Comment by mvarchdev** [comment]
Hello guys! Is there maybe some "native" solution for this, or is it expected to use patches?

**Reply by mvarchdev** [reply]
Ah... Found it here: https://docs.perseides.org/guides/v2/customize-admin-ui/standalone

**Comment by mvarchdev** [comment]
TBH, this looks like bad idea to do... It's like pulling/downloading functionality that you do not need and then explicitly remove it with some patch, not good... I think better is to duplicate admin UI repo and manage it separately, e.g. remove things you do not need, as it can help a lot. What do you think guys?
